phpcs.xml

```diff
@@ -14,6 +14,10 @@
 	<exclude-pattern>./p/scripts/vendor/</exclude-pattern>
 	<exclude-pattern>./vendor/</exclude-pattern>
 	<exclude-pattern>*.min.js$</exclude-pattern>
+	<exclude-pattern>./p/themes/Ansum/*.css$</exclude-pattern>
+	<exclude-pattern>./p/themes/Mapco/*.css$</exclude-pattern>
+	<exclude-pattern>./p/themes/Swage/*.css$</exclude-pattern>
+	<exclude-pattern>*.rtl.css$</exclude-pattern>
```

Comment:
I am not so happy to exclude this files.
The CSS files were changed without the SCSS files in the past, so we should keep the phpcs check for this CSS files.

Reply:
All files excluded here are normally auto-generated, so in a way it does make sense. That being said, what specifically is the reason to exclude these randomly in this PR?

Reply:
That is my fault, because there was several times in a raw fixes to be done in this PR to pass the automated tests, with changes that did not really make sense

Reply:
@math-GH The excluded files are all auto-generated. But I do not mind reverting that change

Reply:
Looking in closer detail, if you're talking about https://github.com/FreshRSS/FreshRSS/pull/4711/commits/1bab465af9715346afd4ce56a049a9edf563dcae then that definitely *does* make sense because frss.rtl.css was out of sync with frss.css. ^_^

It's not quite the correct test or fix but I'd rather keep it in place as an indicator unless someone writes one.

Perhaps it'd be as simple as adding `rtl` to this line:
https://github.com/FreshRSS/FreshRSS/blob/cb1c2da8e6bd923044df3ab1e5fd56cf95e24ef2/Makefile#L225-L226

Reply:
Maybe we could/should automatically call `make fix-all` in PRs:
https://peterevans.dev/posts/github-actions-how-to-automate-code-formatting-in-pull-requests/

Reply:
That might not be a bad idea, although I do worry it could be confusing.

Reply:
I have reverted those changes for now